instant-xml/src/impls.rs

```diff
@@ -270,9 +270,9 @@ impl<'xml> FromXml<'xml> for String {
         }
 
         match deserializer.take_str()? {
-            Some(value) => *into = Some(value.into_owned()),
-            None => return Ok(()),
-        }
+            Some(value) => into.replace(value.into_owned()),
+            None => into.replace(String::new()),
```

Comment:
You've fixed the `String` impl here -- do we need similar fixes for `&str` and `Cow<'_, str>`?

Reply:
Yeah; I've added `Cow<'_, str>` in the latest push. There is no impl for `str`.